default/stringtables/en.txt

```diff
@@ -6238,6 +6241,40 @@ Opinions of Empires:
 OPINIONS_OF_OTHER_SPECIES
 Opinions of Other Species:
 
+# Species Pedia Categories
+
+LITHIC_SPECIES_CLASS
+Lithic Species
+
+LITHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia LITHIC_SPECIES_TITLE]] species:
+
+ORGANIC_SPECIES_CLASS
+Organic Species
+
+ORGANIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ORGANIC_SPECIES_TITLE]] species:
+
+PHOTOTROPHIC_SPECIES_CLASS
+Phototrophic Species
+
+PHOTOTROPHIC_SPECIES_CLASS_DESC
+List of [[encyclopedia PHOTOTROPHIC_SPECIES_TITLE]] species:
+
+ROBOTIC_SPECIES_CLASS
+Robotic Species
+
+ROBOTIC_SPECIES_CLASS_DESC
+List of [[encyclopedia ROBOTIC_SPECIES_TITLE]] species:
+
+SELF_SUSTAINING_SPECIES_CLASS
```

Comment:
Oh, yes, ugly.

To avoid this, is it ok to leave blank the `short_description` value in `default/scripting/encyclopedia/ENC_SHIP_PART.focs.txt `? (as `short_description` is mandatory)
```
Article
    name = "ENC_SHIP_PART"
    category = "ENC_SHIP_PART"
    short_description = "" (currently ENC_SHIP_PART)
    description = "ENC_SHIP_PART_DESC"
    icon = "icons/ship_parts/generic_part.png"
```

If it's ok, I can make a dedicated PR, because there is the same problem in:

```
default/scripting/encyclopedia/ENC_SPECIES.focs.txt
default/scripting/encyclopedia/ENC_SHIP_HULL.focs.txt
default/scripting/encyclopedia/ENC_SHIP_PART.focs.txt
default/scripting/encyclopedia/ENC_TECH.focs.txt
default/scripting/encyclopedia/FIELDS.focs.txt
```